app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/utils/CheckYourBankAccountDetailsHelper.scala

```diff
@@ -31,18 +33,34 @@ class CheckYourBankAccountDetailsHelper @Inject() (implicit langs: Langs, messag
 
   private val key = "bank-details"
 
-  def bankAccountDetailsSummary(displayBankAccountDetails: BankAccount): List[SummaryListRow] =
+  def bankAccountDetailsSummary(
+    displayBankAccountDetails: BankAccountAcc14,
+    router: ReimbursementRoutes
+  ): List[SummaryListRow] =
     List(
-      makeAccountHolderNameRow(displayBankAccountDetails.accountName),
+      makeAccountHolderNameRow(displayBankAccountDetails.accountName, router),
       makeSortCodeRow(displayBankAccountDetails.sortCode),
       makeAccountNumberRow(displayBankAccountDetails.accountNumber)
     ).flattenOption
 
-  private def makeAccountHolderNameRow(accountHolderName: String): Option[SummaryListRow] =
+  private def makeAccountHolderNameRow(accountHolderName: String, router: ReimbursementRoutes): Option[SummaryListRow] =
     Some(
       SummaryListRow(
         Key(Text(messages(s"$key.account-name.label")(lang))),
-        Value(Text(accountHolderName))
+        Value(Text(accountHolderName)),
+        "",
+        Some(
+          Actions(
+            "govuk-link",
+            List(
+              ActionItem(
+                //TODO: Change routing for this link to go to the 'Enter account type' page
```

Comment:
Is this comment still neccessary?

Reply:
Yes, the routing needs to be changed in the next ticket in the bank details journey - CDSR-676